runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,127 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class vim.lsp.inlay_hint.get.opts
+--- @field bufnr integer?
+--- @field filter function?
+--- @field id integer?
+--- @field name string?
+--- @field range lsp.Range?
+---
+--- @class vim.lsp.inlay_hint.get.resp
```

Comment:
Is `resp` a naming convention consistent with existing type annotations?  I saw `ret` was used in other meta types, and personally don't think `resp` (stands for `response`) is a crystal-clear abbreviation. In addition, some other type names starts with `lsp.`.

Reply:
changed to `ret`, thank you for the tip

regarding `vim.lsp.*` vs `lsp.*`: I think using `lsp.*` in `protocol.lua` and `vim.lsp.*` elsewhere would be good for distinguishing between Microsoft's LSP standard and Neovim's LSP client API